### PR TITLE
Add optional section with additional data to html report

### PR DESCRIPTION
**Pull request by lu-maca**
here
As shown in the picture above, I added the possibility to put some additional information to the summary of the report. In particular, these additional info are added for each "behave.additional-info.something" in the [behave.userdata] section in behave.ini. In the example above:

[behave.userdata]
behave.additional-info.tester=lumaca
behave.additional-info.board=multiboard

### CODE REVIEW

File: behave_html_pretty_formatter/html_pretty.py [hunk 1]
```diff
@@ -221,6 +221,20 @@ def generate_feature(self, formatter):
                     with a(onclick="expander('expand_all_failed')", href="#"):
                         div("[Expand All Failed]", cls="feature-summary-row button")
 
+            if formatter.additional_info:
```
Comment (fpokryvk) [inline behave_html_pretty_formatter/html_pretty.py]: Toggle summary will not work for this, if it is not inside `feature-summary-container`. Either add `feature-aditional-info-container` to javascript. Or better, put this block inside `feature-summary-container` (one more indent + additional change to CSS may be needed).

**Review by fpokryvk — CHANGES_REQUESTED** [review]
Thank you for your work on this. There is small thing to change in the code, see inline comment. Please squash the first and the last commit (so unused function is not added at all). Also, after changing CSS or JS, please run `tox -e minify` to compile minified versions and commit them as well.

**Comment by lu-maca** [comment]
Hi @fpokryvk, 
thanks for your comments. I've updated all the stuff, but I'm not sure how to squash all the commits of an already opened pull request. How can I do that without messing up all the work?

**Comment by bittner** [comment]
> I'm not sure how to squash all the commits of an already opened pull request. How can I do that without messing up all the work?

Locally, you would do an interactive rebase (to squash i.e. combine your commits into a single commit) and then force-push the result to this PR.

```console
git rebase -i HEAD~5
```
```console
git push -f
```

Here is some background reading:

- https://www.git-tower.com/learn/git/faq/git-squash/
- https://duckduckgo.com/?t=ffab&q=git+squash&atb=v155-1&iax=videos&ia=videos

**Comment by lu-maca** [comment]
Thanks @bittner for the references!

Now it should be ok

File: behave_html_pretty_formatter/html_pretty.py [hunk 2]
```diff
@@ -221,6 +221,20 @@ def generate_feature(self, formatter):
                     with a(onclick="expander('expand_all_failed')", href="#"):
                         div("[Expand All Failed]", cls="feature-summary-row button")
 
+                if formatter.additional_info:
+                    with div(
+                        cls="feature-additional-info-container",
+                        id="additional-info",
+                        style=f"display: {summary_display}"
```
Comment (fpokryvk) [inline behave_html_pretty_formatter/html_pretty.py]: This line is wrong. It makes additional info never show, if there is "show_summary = false" in config. Please drop this line.

**Review by fpokryvk — CHANGES_REQUESTED** [review]
And final change, in `setup.py` change version to "1.3" and we are good to go. I have run this locally without style="display..." line and seems fine.

File: behave_html_pretty_formatter/html_pretty.py (hunk 2 above)
Comment (fpokryvk) [reply behave_html_pretty_formatter/html_pretty.py]: And also short mention in `CHANGES` what is new in 1.3. Thanks a lot.

**Comment by fpokryvk** [comment]
Check are failing, code should be black formatted, so please delete mentioned `style=f"display..."` line and format by `black`. I can do CHANGES and version increase when I will create the package.

**Comment by fpokryvk** [comment]
Also, please not this issue #9, so there is a chance of repositioning additional data. Plan so far is to make it 3-column flex `div` (scenario counts, expand buttons, time and additional data), that will be responsive for small width too.

**Comment by lu-maca** [comment]
Hi @fpokryvk,

I've fixed that line and reformatted according to black. I leave you all the changes in setup.py/CHANGES :)

**Review by fpokryvk — APPROVED** [review]
Looks good, sorry for initial delay. All checks passed. Merging.

**Comment by fpokryvk** [comment]
And 1.3 is live in pip! Thanks for nice feature!

**Pull request merged**